package.json

```diff
@@ -5,7 +5,9 @@
   "main": "TODO",
   "scripts": {
     "start": "webpack-dev-server --mode development",
-    "build:prod": "webpack --mode production",
+    "build": "webpack --mode production",
```

Comment:
We need to re-add a `build:prod` command for netlify (or update netlify)

Reply:
Good catch.